packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js

```diff
@@ -0,0 +1,290 @@
+// @ts-nocheck
+import {LightningElement, api, track} from 'lwc';
+// @ts-ignore
+import {
+  registerComponentForInit,
+  initializeWithHeadless,
+} from 'c/quanticHeadlessLoader';
+import {
+  NavigationMixin,
+  // @ts-ignore
+} from 'lightning/navigation';
+import {STANDALONE_SEARCH_BOX_STORAGE_KEY, keys} from 'c/quanticUtils';
+
+import search from '@salesforce/label/c.quantic_Search';
+
+const CLASS_WITH_SUBMIT =
+  'slds-combobox__form-element slds-input-has-icon slds-input-has-icon_right slds-input-has-fixed-addon';
+const CLASS_WITHOUT_SUBMIT =
+  'slds-combobox__form-element slds-input-has-icon slds-input-has-icon_left-right';
+
+export default class QuanticStandaloneSearchBox extends NavigationMixin(
+  LightningElement
+) {
+  
+  labels = {
+    search,
+  };
+
+  // Public props
+  /** @type {number} */
+  @api numberOfSuggestions = 5;
+  /** @type {string} */
+  @api placeholder = `${this.labels.search}...`;
+  /** @type {string} */
+  @api engineId;
```

Comment:
there's something weird going on with this file

Reply:
you mean github coloring it? Not much I can do about it ;)